.gitignore

```diff
@@ -6,6 +6,8 @@
 *.dylib
 /inspektor-gadget
 /kubectl-gadget
+/networkpolicyadvisor
```

Comment:
Did you run "go build" manually, or the makefile?

Reply:
You're right, that executable is not generated by "make build". I updated the PR.